middle_end/flambda2/types/env/typing_env.ml

```diff
@@ -1257,6 +1260,53 @@ end = struct
       next_binding_time = env.next_binding_time
     }
 
+  let create_from_closure_conversion_approx
+      (symbols : _ Value_approximation.t Symbol.Map.t) =
+    let defined_symbols_without_equations =
+      Symbol.Map.keys symbols |> Symbol.Set.elements
```

Comment:
This should be `[]`, I think. All symbols are added to the `names_to_types` map, so they all have equations in the result. I'll check that it doesn't obviously break anything, and push the change.

Reply:
Done.